modules/platforms/dotnet/Apache.Ignite.Core.Tests/Compute/ComputeApiTest.cs

```diff
@@ -43,7 +43,10 @@ public partial class ComputeApiTest
         private const string JavaBinaryCls = "PlatformComputeJavaBinarizable";
 
         /** */
-        public const string DefaultCacheName = "default";
+        private const string DefaultCacheName = "default";
+
+        /** */
+        private const string CacheName2 = "cache1";
```

Comment:
"cache2" looks more in sync

Reply:
Cache name is defined in Spring config, let's keep it as is.